lib/core/importer.ts

```diff
@@ -44,21 +43,22 @@ export class Importer {
 
   private scan(pathName: string): AutoClasses {
     const exports: Record<string, unknown> = require(pathName);
-    const autoClasses = Object.values(exports).filter((value) => typeof value === 'function') as ClassType[];
+    return (Object.values(exports) as ClassType[]).reduce(
```

Comment:
combined 'filter' and 'reduce' into a single loop to reduce it to only one iteration ㄹor performance improvement.

Reply:
it's was nice!!!